content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -48,41 +110,39 @@ The following configuration options behave consistently across the latest versio
 : **Supported Input**: A regex string <br>
 : **Description**: Applies a regex to redact sensitive data from query strings on incoming HTTP requests. The default regex matches various sensitive data patterns, including passwords, tokens, API keys, private keys, and authorization terms. Matches are replaced with `<redacted>`. If an empty string is passed, no obfuscation occurs. The resulting value is reported in the `http.url` tag.
 
-### Diagnostics
-
-`DD_TRACE_LOG_DIRECTORY`
-: **Default**: Varies by SDK, environment, and runtime. Please read more in the specific configuration page above for your chosen language <br>
-**Supported Input**: A valid full or relative directory path that exists on the system<br>
-**Caveats**: Not supported in Java, Node.js, Ruby, Python<br>
-**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK falls back to its default diagnostic logging method.
-
-### Agent
 
-`DD_TRACE_AGENT_URL`
-: **Default**: `http://localhost:8126` <br>
-**Supported Input**: A string representing an HTTP or UDS url <br>
-**Description**: The URL for connecting the tracer to the Datadog agent. Valid URL schemas include `http://` and `unix://` (UNIX Domain Sockets). This value takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.
-
-### Unified Service Tagging
+`DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean (`true`/`false`) <br>
+**Description**: Controls whether new traces use 128-bit W3C trace IDs (32-character hexadecimal strings) or 64-bit Datadog trace IDs (16-character hexadecimal strings). The default is `true` to support W3C trace context propagation.
 
-`DD_VERSION`
-: **Default**: `null`<br>
-**Supported Input**: A string representing an application version<br>
-**Caveats**: Node.js defaults to the version number from package.json<br/>
-**Description**: Adds a `version` tag to all spans, except for [Inferred services][3]
+`DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean (`true`/`false`) <br>
+**Caveats**: Not supported in C++<br>
```

Comment:
Available in dd-trace-go v2 only.

Reply:
Do we plan to release v2 soon? If so, we can omit this detail. This page documents supported configurations just for the latest release